src/Nokia_LCD.cpp

```diff
@@ -14,6 +14,12 @@
 namespace {
 const uint8_t kDisplay_max_width = 84;
 const uint8_t kDisplay_max_height = 48;
+
+const LcdFont nokiaFont{(const unsigned char*) Nokia_LCD_Fonts::kDefault_font, 
```

Comment:
Is the cast necessary?
In any case, since we are writing C++ C-style casts should be avoided.